packages/config/src/layer2s/zksyncera.ts

```diff
@@ -73,10 +77,54 @@ export const zksyncera: Layer2 = {
     },
   },
   riskView: makeBridgeCompatible({
-    stateValidation: RISK_VIEW.STATE_ZKP_SN,
-    dataAvailability: RISK_VIEW.DATA_ON_CHAIN,
-    upgradeability: RISK_VIEW.UPGRADABLE_YES,
-    sequencerFailure: RISK_VIEW.SEQUENCER_NO_MECHANISM,
+    stateValidation: {
+      value: 'ZK proofs (SN)',
+      description:
+        'ZK-SNARKS are zero knowledge proofs that ensure state correctness, but require trusted setup.',
+      references: [
+        'https://etherscan.io/address/0x3dB52cE065f728011Ac6732222270b3F2360d919#code#F5#L89',
+        'https://etherscan.io/address/0x389a081BCf20e5803288183b929F08458F1d863D#code#F10#L254',
+        'https://etherscan.io/address/0xF1fB730b7f8E8391B27B91f8f791e10E4a53CEcc#code#F7#L24',
+        'https://etherscan.io/address/0x473b1887d45d61efd87731a1d8ec3590b93c565d#code#F5#L227',
+        'https://era.zksync.io/docs/dev/developer-guides/transactions/transactions.html#transaction-types',
+        'https://era.zksync.io/docs/dev/developer-guides/system-contracts.html#executorfacet',
+      ],
+      contracts: ['ValidatorTimelock', 'DiamondProxy', 'Verifier'],
+    },
```

Comment:
WYT about spreading value and description from a common file and adding references on top of that? This will prevent us from typos and differing descriptions for the same risks